Makefile

```diff
@@ -107,120 +111,100 @@ configure-installer-tests-cluster:
 
 # Run odo app cmd tests
 .PHONY: test-cmd-app
 test-cmd-app:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odoCmdApp" \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) -focus="odoCmdApp" tests/integration/ -timeout $(TIMEOUT)
 
 # Run login e2e tests
 # This test shouldn't run spec in paralel because it will break the test behaviour
 # due to race condition in parallel run.
 .PHONY: test-odo-login-e2e
 test-odo-login-e2e:
-	ginkgo -v -nodes=1 -focus="odoLoginE2e" \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/loginlogout/ -timeout $(TIMEOUT)
+	ginkgo -v -nodes=1 -focus="odoLoginE2e" tests/integration/loginlogout/ -timeout $(TIMEOUT)
 
 # Run config tests
 .PHONY: test-odo-config
 test-odo-config:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odo config test" \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) -focus="odo config test" tests/integration/ -timeout $(TIMEOUT)
 
 # Run command's integration tests irrespective of service catalog status in the cluster.
 # Service, link and login/logout command tests are not the part of this test run
 .PHONY: test-integration
 test-integration:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) tests/integration/ -timeout $(TIMEOUT)
 
 # Run command's integration tests which are depend on service catalog enabled cluster.
 # Only service and link command tests are the part of this test run
 .PHONY: test-integration-service-catalog
 test-integration-service-catalog:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/servicecatalog -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) tests/integration/servicecatalog -timeout $(TIMEOUT)
 
 # Run url integreation tests
 .PHONY: test-odo-url-int
 test-odo-url-int:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odoURLIntegration" \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) -focus="odoURLIntegration" tests/integration/ -timeout $(TIMEOUT)
 
 # Run push command e2e
 .PHONY: test-cmd-push
 test-cmd-push:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odo push command tests" \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) -focus="odo push command tests" tests/integration/ -timeout $(TIMEOUT)
 
 # Run e2e test scenarios
 .PHONY: test-e2e-scenarios
 test-e2e-scenarios:
-	ginkgo -v -nodes=$(TEST_EXEC_NODES) -randomizeAllSpecs \
-	slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) tests/e2escenarios/ -timeout $(TIMEOUT)
+	ginkgo $(GINKGO_FLAGS) -randomizeAllSpecs \
```

Comment:
Since GINKGO_FLAGS seems to contain the common flags, do we need them?

Reply:
Technically no, but I didn't want to break existing use of those variables and surprise people.

Reply:
Let's remove them as it might create confusion later